Login and Signup

Vyshnav-KS opened the pull request.

SuvarneshKM committed:
Create README.md

Vyshnav-KS committed:
login/signup mob

Vyshnav-KS committed:
login/signup

Vyshnav-KS committed:
login/signup

Vyshnav-KS committed:
login

Vyshnav-KS committed:
login/signup links

superRaptor911 commented:
Use png instead of svg, svg's are bigger in size.

superRaptor911 commented:
Try to fix codefactor errors
[https://www.codefactor.io/repository/github/superraptor911/expensemeter/pull/8](https://www.codefactor.io/repository/github/superraptor911/expensemeter/pull/8)

superRaptor911 committed:
Empty gatsby project with eslint and aphrodite

superRaptor911 committed:
Main Page Desktop

superRaptor911 committed:
Fixed button border

superRaptor911 committed:
Refactored Code

superRaptor911 committed:
Title component

superRaptor911 committed:
Things we do

superRaptor911 committed:
Things we do

superRaptor911 committed:
bg image auto scale and scroll to our vision

superRaptor911 committed:
Title text stroke

superRaptor911 committed:
Our initiative component

ImgBotApp committed:
[ImgBot] Optimize images

*Total -- 2,488.34kb -> 2,280.29kb (8.36%)

/src/images/icons/heartIcon.png -- 2.65kb -> 1.42kb (46.39%)
/src/images/icons/cpuIcon.png -- 2.02kb -> 1.10kb (45.41%)
/src/images/icons/flagIcon.png -- 1.99kb -> 1.12kb (43.68%)
/src/images/hackClubFlag.png -- 14.68kb -> 11.49kb (21.72%)
/src/images/icons/twitter.png -- 0.79kb -> 0.67kb (15.12%)
/src/images/icons/github.png -- 0.80kb -> 0.70kb (12.01%)
/src/images/icons/linkedin.png -- 0.72kb -> 0.64kb (10.5%)
/src/images/icons/instagram.png -- 0.76kb -> 0.69kb (9.41%)
/src/images/bgDesktop.png -- 1,960.20kb -> 1,781.43kb (9.12%)
/src/images/iphone.png -- 304.15kb -> 286.10kb (5.93%)
/src/images/ihih.png -- 199.58kb -> 194.93kb (2.33%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>

superRaptor911 committed:
Our initiative component

superRaptor911 committed:
Upcoming events

superRaptor911 committed:
Smooth Scrolling

superRaptor911 committed:
Merge pull request #1 from superRaptor911/imgbot

[ImgBot] Optimize images

superRaptor911 committed:
Gallery

superRaptor911 committed:
Merge branch 'main' of https://github.com/superRaptor911/v4

superRaptor911 committed:
Gallery images

superRaptor911 committed:
Gallery animations

superRaptor911 committed:
Footer

superRaptor911 committed:
Gallecry Images indicator

superRaptor911 committed:
Footer modification

superRaptor911 committed:
Footer modification, for akhil tj and aditya aravind credits

superRaptor911 committed:
Responsive header

superRaptor911 committed:
Fixed 13688x768 display bug

superRaptor911 committed:
More responsive header and upcoming events

superRaptor911 committed:
Hackclub flag fix

superRaptor911 committed:
Minor header changes

Vyshnav-KS committed:
started

Vyshnav-KS committed:
started

Vyshnav-KS committed:
Main Page Responsive

Vyshnav-KS committed:
lock files changed

Vyshnav-KS committed:
lock files changed

Vyshnav-KS committed:
yarn

Vyshnav-KS committed:
Responsive view main

Vyshnav-KS committed:
bug

superRaptor911 committed:
Merge pull request #5 from Vyshnav-KS/main

Main Page Responsive

ImgBotApp committed:
[ImgBot] Optimize images

*Total -- 254.82kb -> 235.05kb (7.76%)

/src/images/icons/hackclubFlagIcon.png -- 3.82kb -> 2.05kb (46.45%)
/src/images/bgMobile.png -- 251.00kb -> 233.01kb (7.17%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>

superRaptor911 committed:
Fixed SSR bug

superRaptor911 committed:
useDeviceType Hook

superRaptor911 committed:
Merge pull request #6 from superRaptor911/imgbot

[ImgBot] Optimize images

superRaptor911 committed:
Desktop Bg image optimisation

superRaptor911 committed:
Merge branch 'main' of https://github.com/superRaptor911/v4

superRaptor911 committed:
Mobile Bg image optimisation

superRaptor911 committed:
Mobile Bg image optimisation

superRaptor911 committed:
Fixed x overflow

Vyshnav-KS committed:
ssr bug

superRaptor911 commented:
da ith enth?
